servicetalk-utils-internal/src/main/java/io/servicetalk/utils/internal/PlatformDependent0.java

```diff
@@ -142,37 +146,56 @@ final class PlatformDependent0 {
                         MethodType.methodType(void.class, Object.class),
                         throwExceptionMH,
                         MethodType.methodType(void.class, Throwable.class));
-                throwConsumer = (Consumer<Throwable>) throwExceptionCallSite.getTarget().bindTo(unsafe).invoke();
-            } catch (Throwable all) {
-                LOGGER.warn("failed computing THROW_EXCEPTION", all);
+                //noinspection unchecked
+                Consumer<Throwable> unsafeThrowConsumer =
+                        (Consumer<Throwable>) throwExceptionCallSite.getTarget().bindTo(unsafe).invoke();
+                throwConsumer = (t) -> {
+                    try {
+                        // JVM has been observed to crash when passing a null argument.
+                        // See https://github.com/netty/netty/issues/4131.
+                        unsafeThrowConsumer.accept(null != t ? t : new NullPointerException("Throwable was null"));
+                    } catch (Throwable all) {
+                        throwException0(t);
```

Comment:
This is likely a problem. `unsafeThrowConsumer.accept(...)` always throws. Then we will catch it and rethrow again without unsafe. No benefit of using unsafe then.

Reply:
Hah, yes… What do you think about removing the catch?

Reply:
yes, we have to remove the `try-catch` block

Reply:
done.